pandas/tests/arrays/masked/test_arrow_compat.py

```diff
@@ -39,6 +55,28 @@ def test_arrow_roundtrip(data):
     tm.assert_frame_equal(result, df)
 
 
+@td.skip_if_no("pyarrow")
+def test_dataframe_from_arrow_type_mapper(int_dtype):
+    pyarrow = pytest.importorskip("pyarrow", minversion="3.0.0")
```

Comment:
put the version IN the skip_if_no

Reply:
can you add a reference to this issue in a comment (e.g. the gh number)

Reply:
Turns out the module-level limit of 1.0.1 seems to be OK. Just tested with that version.